Example/Tests/Spec Helpers/SpecUtils.swift

```diff
@@ -79,7 +79,8 @@ struct SpecUtils {
             "status": status
         ]
 
-        return SEEncryptedAuthorizationData(dict)!.decryptedAuthorizationDataV2!
+        let response = SpecDecodableModel<SEEncryptedAuthorizationData>.create(from: dict)
+        return (response?.decryptedAuthorizationDataV2!)!
```

Comment:
I don't get why you remade this one. Nothing changed as a result.

Reply:
I did it so that I could work with encrypted data from `dict`.

Reply:
For here ok. Rewrite it like this:

```
let response = SpecDecodableModel<SEEncryptedAuthorizationData>.create(from: dict)!
return response.decryptedAuthorizationDataV2!
```